lib/content-services/src/lib/i18n/ar.json

```diff
@@ -1,4 +1,8 @@
 {
+  "COMMON": {
```

Comment:
We already have them, I did it just to have something common, not specific. Should I remove it anyway?

Reply:
I also plan to create a story on replacing the remaining buttons with COMMON, since the project should basically only have one translation for “confirm” and one for “cancel”.

Reply:
here is a story https://hyland.atlassian.net/browse/ACS-7959

Reply:
Okay if they we're already translated properly then it's okay for me